### PR TITLE
Updated with CAP 1.4 announcement

### PR DESCRIPTION
**Pull request by gaktive**

### CODE REVIEW

File: adoc/Release-Notes.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ifdef::env-github[]
 :suse: SUSE
 :product: {suse} Cloud Application Platform
-:version: 1.3.1
+:version: 1.4
```
Comment (ghost) [inline adoc/Release-Notes.adoc:5]: This value will be ignored in the published document -- please fix the value in `adoc/entities.adoc` and then run `make update-entries`.

File: adoc/Release-Notes.adoc [hunk 2]
```diff
@@ -31,6 +31,112 @@ endif::[]
...
+
+* If autoscaler was enabled in {product} 1.3.1, you must specify `sizing.autoscaler_postgres.disk_sizes.postgres_data=100` during the helm upgrade to avoid upgrade errors. Alternatively, you can disable the autoscaler before the upgrade and re-enable after the upgrade is finished. Without any of these workarounds, the upgrade would fail with `Error: UPGRADE FAILED: StatefulSet.apps "autoscaler-postgres" is invalid` message.
+
+* If you are using the NGINX Ingress Controller and seeing `Request Entity Too Large` errors, you should bump up the ingress proxy body size to an appropriate value by setting the `ingress.annotations` key in helm chart values as in the following:
+
+```
+  ingress:
+     annotations:
+       nginx.ingress.kubernetes.io/proxy-body-size: 64m
+```
```
Comment (ghost) [inline adoc/Release-Notes.adoc:131]: This screen is formatted like Markdown, and won't work in AsciiDoc. It should look like:
 
```
+
[source]
----
  ingress:
     annotations:
       nginx.ingress.kubernetes.io/proxy-body-size: 64m
----
```

Note the `+` in the first line (i.e. replacement for empty line 126), that will keep the screen within the list indentation.

**Review by satadruroy — APPROVED** [review]
LGTM

File: adoc/Release-Notes.adoc [hunk 3]
```diff
@@ -31,6 +31,112 @@ endif::[]
...
 == Major Changes
 
+[id='sec.1_4']
+=== Release 1.4, May 2019
+
+[id='sec.1_4.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.16.4:
+** A tech preview of Eirini is available.
```
Comment (troytop) [inline adoc/Release-Notes.adoc:41]: Link to https://github.com/SUSE/scf/wiki/Eirini from here as well (not just from known issues).

File: adoc/Release-Notes.adoc [hunk 4]
```diff
@@ -31,6 +31,112 @@ endif::[]
...
+** python-buildpack: 1.6.32
+** ruby-buildpack: 1.7.38
+** staticfile-buildpack: 1.4.42
+
+[id='sec.1_4.issue']
+==== Known Issues
+
+* The instructions for enabling Eirini can be found at https://github.com/SUSE/scf/wiki/Eirini.
+
+* Resuming a past practice, with {product} 1.4, use the complete command: `helm upgrade --force --recreate-pods` for an upgrade. This will reintroduce downtime for apps but without `--recreate-pods`, multiple versions of statefulsets may co-exist which can cause incompatibilities between depedent statefulsets, and result in a broken upgrade. This applies to Stratos pods as well.
```
Comment (troytop) [inline adoc/Release-Notes.adoc:119]: Offer instructions on mitigating downtime by pre-pulling the CAP 1.4 base images for scf and uaa to each Kubernetes node before running `helm upgrade ...`

**Review by troytop — APPROVED** [review]
These are not blockers to a release announcement. They can be fixed later.

File: adoc/Release-Notes.adoc (hunk 1 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc:5]: Done via https://github.com/SUSE/cap-release-notes/pull/64

File: adoc/Release-Notes.adoc (hunk 2 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc:131]: Done in https://github.com/SUSE/cap-release-notes/pull/61

File: adoc/Release-Notes.adoc (hunk 3 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc:41]: Done in https://github.com/SUSE/cap-release-notes/pull/61

File: adoc/Release-Notes.adoc (hunk 4 above)
Comment (gaktive) [reply adoc/Release-Notes.adoc:119]: Need to do still in a new PR. According to Troy: 
`docker pull registry.suse.com/cap/scf-api-group:c4e92661127f706c3afbceeac23bd8e78349d4ab` 
and `docker pull registry.suse.com/cap/uaa-uaa:378dee28c60e7a7ac892d79ad8fa043d63c2c7cf`

**Pull request merged**